db/migrate/20220831065513_add_track_to_exercise_representations.rb

```diff
@@ -0,0 +1,10 @@
+class AddTrackToExerciseRepresentations < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :exercise_representations, :track, null: true, foreign_key: true, if_not_exists: true
+
+    unless Rails.env.production?
+      exercise_track_id_sql = Arel.sql(Exercise.where('id = exercise_id').select(:track_id).to_sql)
+      Exercise::Representation.update_all("track_id = (#{exercise_track_id_sql})")
+    end
```

Comment:
This should have the logic from the one above (I actually don't think the one above necessarily needs the looping - it was this one I intended it for!)